ui/src/variables/utils/getWindowVars.ts

```diff
@@ -51,6 +51,9 @@ export const getWindowPeriod = (
   query: string,
   variables: VariableAssignment[]
 ): number | null => {
+  if (query === undefined || query.length === 0) {
```

Comment:
It makes me nervous that I don't know if and when query can be undefined here. 
I think we should only return null if `(!query.length)`